docs/docs/integrations/vectorstores/cratedb.ipynb

```diff
@@ -0,0 +1,499 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CrateDB\n",
+    "\n",
+    "This notebook shows how to use the CrateDB vector store functionality around\n",
+    "[`FLOAT_VECTOR`] and [`KNN_MATCH`]. You will learn how to use it for similarity\n",
+    "search and other purposes.\n",
+    "\n",
+    "It supports:\n",
+    "- Similarity Search with Euclidean Distance\n",
+    "- Maximal Marginal Relevance Search (MMR)\n",
+    "\n",
+    "## What is CrateDB?\n",
+    "\n",
+    "[CrateDB] is an open-source, distributed, and scalable SQL analytics database\n",
+    "for storing and analyzing massive amounts of data in near real-time, even with\n",
+    "complex queries. It is PostgreSQL-compatible, based on [Lucene], and inherits\n",
+    "the shared-nothing distribution layer of [Elasticsearch].\n",
+    "\n",
+    "This example uses the [Python client driver for CrateDB]. For more documentation,\n",
+    "see also [LangChain with CrateDB].\n",
+    "\n",
+    "\n",
+    "[CrateDB]: https://github.com/crate/crate\n",
+    "[Elasticsearch]: https://github.com/elastic/elasticsearch\n",
+    "[`FLOAT_VECTOR`]: https://crate.io/docs/crate/reference/en/master/general/ddl/data-types.html#float-vector\n",
+    "[`KNN_MATCH`]: https://crate.io/docs/crate/reference/en/master/general/builtins/scalar-functions.html#scalar-knn-match\n",
+    "[LangChain with CrateDB]: /docs/extras/integrations/providers/cratedb.html\n",
+    "[Lucene]: https://github.com/apache/lucene\n",
+    "[Python client driver for CrateDB]: https://crate.io/docs/python/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Getting Started"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [],
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Install required packages: LangChain, OpenAI SDK, and the CrateDB Python driver.\n",
+    "!pip install 'langchain[cratedb,openai]'"
```

Comment:
That's no longer a valid installation incantation. Something like this would be better:

```suggestion
    "!pip install langchain langchain-openai sqlalchemy-cratedb"
```